RevenueCatUI/PaywallView.swift

```diff
@@ -80,31 +94,73 @@ public struct PaywallView: View {
     public init(
         offering: Offering,
         fonts: PaywallFontProvider = DefaultPaywallFontProvider(),
-        displayCloseButton: Bool = false
+        displayCloseButton: Bool = false,
+        performPurchase: PerformPurchase? = nil,
+        performRestore: PerformRestore? = nil
     ) {
+        let purchaseHandler = PurchaseHandler.default(performPurchase: performPurchase, performRestore: performRestore)
         self.init(
             configuration: .init(
                 offering: offering,
                 fonts: fonts,
-                displayCloseButton: displayCloseButton
+                displayCloseButton: displayCloseButton,
+                purchaseHandler: purchaseHandler
             )
         )
     }
 
+    // Notes: Should this be internal, and public only for dev? Should configuration be totally private, yes?
+    // And these purchase blocks should ONLY be defined on the purchase handler?
```

Comment:
Should we remove this note before merging?

Reply:
Oops yes I never should have checked that in!